httpie/client.py

```diff
@@ -108,24 +108,35 @@ def finalize_headers(headers):
 
 
 def get_default_headers(args):
-    write_info('get_default_headers', 'Total: 3')
+    write_info('get_default_headers', 'Total: 7')
     default_headers = {
         'User-Agent': DEFAULT_UA
     }
 
     auto_json = args.data and not args.form
```

Comment:
Should we add an if-statement for this?

Reply:
I'm not sure, is it only the AND of two bools? If so, I don't think it's necessary since it's a single instruction.

Reply:
I think so, let's skip it then.